cgroups/systemd/apply_systemd.go

```diff
@@ -212,6 +213,17 @@ func (m *Manager) Apply(pid int) error {
 
 	m.Paths = paths
 
+	cpuShares, err := ioutil.ReadFile(path.Join(m.Paths["cpu"], "cpu.shares"))
+	if err != nil {
+		return err
+	}
+
+	if c.CpuShares != 0 {
+		if strings.TrimSpace(strconv.FormatInt(c.CpuShares, 10)) != strings.TrimSpace(string(cpuShares)) {
```

Comment:
I just tried, `fmt.Sprintf()`, It is printing nothing on docker daemon logs or client, which I was expecting. we need to throw the error back from libcontainer to docker. Let me know if this works for you.

I ll take a look at FScanf and see if I can read `in64` directly from the file.